.github/workflows/run-tests.yml

```
@@ -0,0 +1,48 @@
+name: Run Backend Unit & Integration Tests
+
+on:
+  pull_request:
+    branches: [ develop ]
```

```suggestion
    branches: [ develop, master ]
```
Actually I forgot about this, just to be sure we should also add this action to PR's to Master (for release)